codespell_lib/tests/test_dictionary.py

```diff
@@ -75,6 +76,9 @@ def _check_aspell(phrase, msg, in_aspell, fname):
         return  # stop normal checking as we've done each word above
     this_in_aspell = speller.check(
         phrase.encode(speller.ConfigKeys()['encoding'][1]))
+    if not this_in_aspell:
+        this_in_aspell = spellerGB.check(
+            phrase.encode(spellerGB.ConfigKeys()['encoding'][1]))
```

Comment:
I guess you _could_ also do:
```
this_in_aspell = speller.check(phrase.encode(speller.ConfigKeys()['encoding'][1])) or spellerGB.check(phrase.encode(spellerGB.ConfigKeys()['encoding'][1]))
```
or even:
```
this_in_aspell = any(sp.check(phrase.encode(sp.ConfigKeys()['encoding'][1])) for sp in (speller, spellerGB))
```
:wink:

Reply:
I "improved" it further.